modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendarAccessibility.testcase

```diff
@@ -76,4 +76,33 @@ definition {
 		}
 	}
 
+	@description = "LPS-185089 Verifies that the focus is not lost after closing the 'Edit Event' modal."
+	@priority = 3
+	test FocusIsNotLostAfterClosingTheModal {
+		SetWindowSize(value1 = "1920,1540");
+
+		task ("And a new event is added") {
+			CalendarNavigator.gotoAddEvent();
+		}
+
+		task ("When the user tries to edit the event") {
+			CalendarEvent.editTitle(eventTitle = "Calendar Event Title");
+		}
+
+		task ("And clicks any 'Select' button on 'CATEGORIZATION'") {
+			Click(locator1 = "Button#SELECT_CATEGORIES");
+		}
```

Comment:
Expand the categorization panel before clicking

Reply:
Interacted elements need to actually visible before clicking to mimic user experience. Expanding the panel also would allow the screenshot a chance to capture the focus visible area.

Reply:
Right... I'll try it

Reply:
Even after expanding CATEGORIZATION, it seems the `Select` button is not visible...

Reply:
weird...

Reply:
Right... I'll try to scroll to an element below that one, so the `Select` button should be visible